src/include/duckdb/common/types/vector.hpp

```diff
@@ -35,6 +35,11 @@ struct UnifiedVectorFormat {
 	}
 };
 
+struct RecursiveUnifiedVectorFormat {
+	UnifiedVectorFormat format;
```

Comment:
I found this name confusing when reading the rest of the code. Can we call them something like `unified` and `children`?